### ISSUE TITLE
Is numexpr resally necessary?

### ISSUE THREAD

**Issue by barkls** [phys201/extended-beam-modeling#20]
I might be wrong, but it looks like it's only used to convert date/times into coordinates, which shouldn't be a very expensive operation, so numpy should be sufficient

**Comment by barkls** [comment]
Right now numexpr is used every time you call `Model.observe`, but I think this can be precalculated. See #21.

**Comment by jacornelison** [comment]
This is true. The code that converts topocentric to celestial coordinates comes from an external source so I wanted to change as little as possible to it. But as you say, `numexpr` is a bit superfluous, so I'll go ahead and change the code over.

**Comment by barkls** [comment]
Just a note this shouldn't be a priority since the current code works, so you might be better off focusing on getting inference up-and-running or writing more tests.
